templates/6_tma_batch_qc_metrics.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<p>I think it would be simpler to provide a list of FOVs to plot, rather than substrings. The user can generate the list however they want. But not everyone will have this same format, so I think having more flexibility would be better. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>